### DISCUSSION TITLE
In-game markdown doesn't understand inline code blocks

### DISCUSSION THREAD

**Discussion by lilyremigia** [ppy/osu#15729] [Ideas] [answered]
In a nutshell support for `these`.

See in-game changelog
![in-game changelog](https://user-images.githubusercontent.com/13798839/142785014-90e7884b-4d02-4f1f-9a1e-cc43b845c068.png) 
vs. the changelog on the web:
![changelog on the web](https://user-images.githubusercontent.com/13798839/142785078-f76a57d0-d608-48cb-b0da-68315a6bbbe5.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15730
